File: src/arcade/intro/theJourneyBegins/CenturyFromYear.java
package arcade.intro.theJourneyBegins;

public class CenturyFromYear {
    public static void main(String[] args) {
        System.out.println(centuryFromYear(2021));
    }

    public static int centuryFromYear(int year) {
        if (year % 100 != 0)
            return year / 100 + 1;
        else
            return year / 100;
    }
}

File: src/arcade/intro/theJourneyBegins/CenturyFromYear.kt
package arcade.intro.theJourneyBegins

fun main() {
    println("Century from year : ${solution(1700)}")
}

fun solution(year: Int): Int {
    if (year % 100 != 0)
        return year / 100 + 1
    else
        return year / 100
}
